component-library/component-lib/src/lib/form-components/input/input.component.html

```diff
@@ -52,7 +52,7 @@
             tabindex="0"
             class="passwordIcon"
             (click)="hideShow()"
-            [disabled]="disabled"
+            [disabled]="disabled || config.formGroup.get(config.id)?.disabled"
```

Comment:
Tie this to the disabled variable in the TS instead. This will (likely) run a check every tick. Instead, whenever disabled is set, just have it hook into that. There's a few ways you can do that. Touch base tomorrow afternoon and we can go over them